### PR TITLE
Adjusted logging in EmailSender

### PR DESCRIPTION
**Pull request by jerbri**

### CODE REVIEW

**Comment by mattgwagner** [comment]
Holy in need of a rebase :-)

File: CertiPay.Common.Notifications/Notifications/IEmailService.cs [hunk 1]
```diff
@@ -127,9 +130,10 @@ public void Send(MailMessage message)
             FilterRecipients(message.CC);
             FilterRecipients(message.Bcc);
 
-            Log.Info("Sending email {@message}", message);
-
-            await _smtp.SendMailAsync(message);
+            await _smtp.SendMailAsync(message).ContinueWith(x =>
```
Comment (mattgwagner) [inline CertiPay.Common.Notifications/Notifications/IEmailService.cs]: The `ContinueWith` method has a second parameter, `ContinuationOptions` or similar. We might want to use that to only log the "Sent" message if it's successful here?

Comment (jerbri) [inline CertiPay.Common.Notifications/Notifications/IEmailService.cs]: That was the intended behavior. I'll review the overloads.

**Comment by mattgwagner** [comment]
I cherry-picked the changes for the logging in and updated to use the completion overload mentioned. 

I didn't include the envutil change yet.

**Comment by mattgwagner** [comment]
Actually, the `EnvUtil` was just a casualty of the merge, anyway.

**Pull request closed without merging**